# Array of person positions
std_msgs/Header header
Person[] people


